config/default.json.example

```diff
@@ -13,13 +13,14 @@
     "password": "rogierisgaaf",
     "host": "192.168.99.1",
     "database": "imageresizer"
-  },  
+  },
   "allow_indexing": false,
   "constraints": {
     "max_width": 2000,
     "max_height": 2000
   },
   "log": {
     "debug": false
-  }
+  },
+  "redirect_timeout" : 604800
```

Comment:
Add the parameter description to the readme as well (I think the others are already in there)